openedx_external_enrollments/external_enrollments/icc_external_enrollment.py

```diff
@@ -185,8 +186,15 @@ def _get_icc_user_from_xml_response(self, response, method_type):
                 if field.get('@name') == 'username':
                     icc_user['username'] = field.get('VALUE')
                     break
-        except Exception:  # pylint: disable=broad-except
-            pass
+        except (TypeError, KeyError):  # pylint: disable=broad-except
```

Comment:
Please, move the broad-except flag to the correct exception. @ivanvgh

Reply:
You are right, Done.